pkg/deprecation/deprecation.go

```diff
@@ -43,6 +47,11 @@ var messages = map[Warning]string{
 		"Use `ImagePullSecrets` instead.",
 	CRIRegistryConfigs: "The `configs` property of `[plugins.\"io.containerd.grpc.v1.cri\".registry]` is deprecated since containerd v1.5 and will be removed in containerd v2.0." +
 		"Use `config_path` instead.",
+
+	TracingOTLPConfig: "The `otlp` property of `[plugins.\"io.containerd.tracing.processor.v1\".otlp]` is deprecated since containerd v1.6 and will be removed in containerd v2.0." +
+		"Use OTLP environment variables instead: https://opentelemetry.io/docs/specs/otel/protocol/exporter/",
+	TracingServiceConfig: "The `tracing` property of `[plugins.\"io.containerd.internal.v1\".tracing]` is deprecated since containerd v1.6 and will be removed in containerd v2.0." +
+		"Use OTEL environment variables instead: https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/",
```

Comment:
`RELEASES.md` has to be updated

Reply:
https://github.com/containerd/containerd/blame/main/RELEASES.md#L426
something like this:

|`[plugins.\"io.containerd.tracing.processor.v1.otlp\"]`                      | `endpoint`, `protocol`, `insecure`            | containerd v1.6     | containerd v2.0            | Instead use OTLP environment variables OTEL_EXPORTER_OTLP_TRACES_ENDPOINT, OTEL_EXPORTER_OTLP_PROTOCOL, OTEL_SDK_DISABLED    |

|`[plugins.\"io.containerd.internal.v1.tracing\"]`                      | `service_name`, `sampling_ratio`            | containerd v1.6     | containerd v2.0            | Instead use OTLP environment variables OTEL_SERVICE_NAME, OTEL_TRACES_SAMPLER*            |

Reply:
Added